api/mfx_dispatch/linux/Android.mk

```diff
@@ -5,9 +5,9 @@ LOCAL_PATH:= $(call my-dir)
 include $(CLEAR_VARS)
 include $(MFX_HOME)/android/mfx_defs.mk
 
-LOCAL_SRC_FILES := $(addprefix src/, \
+LOCAL_SRC_FILES := \
```

Comment:
This one is obvious. We can merge this in almost immediately if this will be separated out from the below plugins change...